### PR TITLE
ENH: Add cxx test

### PR DESCRIPTION
**Pull request by tbirdso**

### CODE REVIEW

File: itk-module.cmake [hunk 1]
```diff
@@ -16,6 +16,7 @@ itk_module(Mesh3DProcrustesAlignFilter
     ITKCommon
     ITKMesh
     ITKTransform
+    ITKIOMeshBase
```
Comment (thewtex) [inline itk-module.cmake]: This can go in `TEST_DEPENDS`.

**Review by thewtex — CHANGES_REQUESTED** [review]
👍 One minor tweak, and good to go.

**Comment by tbirdso** [comment]
Rebased on `main`.

**Review by thewtex — APPROVED** [review]
🐰 🚀

**Pull request merged**